packages/storage/src/file.js

```diff
@@ -729,6 +729,13 @@ File.prototype.createResumableUpload = function(options, callback) {
  * Resumable uploads are automatically enabled and must be shut off explicitly
  * by setting `options.resumable` to `false`.
  *
+ * <p class="notice">
+ *   Resumable uploads have been known to cause performance issues. For each
+ *   file being uploaded we store the first 16kb of data, this can cause
+ *   considerable performance degredation when uploading a large number of
+ *   files.
+ * </p>
```

Comment:
Why do we want to note that it only happens to small files? Both of the issues reported mention medium sized files - and based on what I read I thought this issue pertained the volume of images and not the image sizes themselves.